### DISCUSSION TITLE
【求助】请问绳网有发布内容限制吗？

### DISCUSSION THREAD

**Discussion by Oaqn** [share121/inter-knot#1767] [其他]
是只能发和米哈游及旗下游戏相关的内容，还是说其他的也可以发（指健康文明得体规范无不良倾向的内容）
![19a46f1327c84badb7dbd8272b7d9c3b](https://github.com/user-attachments/assets/7bb528bc-9a0d-46ff-a442-1a759c2df363)

**Comment by whsjjj28288** [comment]
你不仅可以发米哈游旗下游戏的相关内容，你还可以讨论其他的，比如说你的学业，或者说有关别的问题，反正只要你能想到啥，就都可以，只要是健康文明的内容

**Comment by SednaMi** [comment]
具体看GitHub 可接受使用政策

**Comment by Oaqn** [comment]
谢谢，还有就是，我是在B站看见作者视频然后下载github来的，请问怎么更改头像
![Uploading 8cd03c58e8240862ac229151d9cdd28f.jpg…]()

**Comment by Oaqn** [comment]
会了，谢谢

**Comment by Kujojolen** [comment]
拜托，这可是绳网欸，By the way What foger is 米哈游？？？？

**Comment by yys20071108** [comment]
不是特别的限制

**Comment by AAAduck** [comment]
基本**free**

**Comment by XokoukiguraX** [comment]
不能引战带节奏

**Comment by ZhuYuan-148** [comment]
严禁辱骂引战赌博，以及，不要搞色色。
